probe/kubernetes/controls.go

```diff
@@ -474,6 +475,13 @@ func (r *Reporter) CaptureJob(f func(xfer.Request, string, string) xfer.Response
 	}
 }
 
+// CaptureCordonNode is exported for testing
+func (r *Reporter) CaptureCordonNode(f func(xfer.Request, string) xfer.Response) func(xfer.Request) xfer.Response {
+	return func(req xfer.Request) xfer.Response {
+		return f(req, r.nodeName)
```

Comment:
Not sure this will work - the node to cordon should come in the request.
(`nodeName` is blank in the recommended Kubernetes configuration for Scope)

Reply:
Sort of - the `Node` there is not a Kubernetes Node, but refers to nodes in the Scope report "graph" - like the mathematical idea of a graph with nodes and edges. 

Anyway, it should be a value you can call `ParseHostNodeID()` on, and get back the hostID, which I was saying above could be set to the Kubernetes `spec.nodeName`.  Look at the way `CapturePod()` works, and you should be able to code `CaptureNode()` similarly.

Reply:
Ah, I understand it now. Thanks for the explanation. Will let you know if I have more doubts.

Reply:
Do I also need to implement a `WalkNodes()` similar to what we have for pods and deployments?

Reply:
That looks likely, to be consistent.

Reply:
Actually I’m not sure. If you have the name of a k8s object you can access it directly.